examples/server/server.cpp

```diff
@@ -453,23 +487,33 @@ int main(int argc, char ** argv) {
         std::vector<float> pcmf32;               // mono-channel F32 PCM
         std::vector<std::vector<float>> pcmf32s; // stereo-channel F32 PCM
 
-        // write to temporary file
-        const std::string temp_filename = "whisper_server_temp_file.wav";
-        std::ofstream temp_file{temp_filename, std::ios::binary};
+        // write file to temporary file
+        std::ofstream temp_file{filename, std::ios::binary};
         temp_file << audio_file.content;
         temp_file.close();
 
+        // if file is not wav, convert to wav
+        if(sparams.ffmpeg_converter) {
+            std::string converted_filename = "converted_" + filename;
+            // ffmpeg command
+            std::string ffmpeg_cmd = "ffmpeg -i " + filename + " -ar 16000 -ac 1 -c:a pcm_s16le " + converted_filename + " 2>&1"; // Redirect stderr to stdout
```

Comment:
Would be cleaner to put this cmd in the executeConvertToWAV function, since it's not used anywhere else here. Just pass the filename to the function.